patroni/api.py

```diff
@@ -89,8 +90,17 @@ def do_GET(self, write_status_code_only=False):
         patroni = self.server.patroni
         cluster = patroni.dcs.cluster
 
-        replica_status_code = 200 if not patroni.noloadbalance and \
-            response.get('role') == 'replica' and response.get('state') == 'running' else 503
+        replica_status_code = 503
+        if response.get('role') == 'replica' and response.get('state') == 'running' \
+           and not patroni.noloadbalance:
+            max_replica_lag = self.path_query.get('lag') and parse_int(self.path_query.get('lag')[0]) or 0
+            leader_optime = cluster and cluster.last_leader_operation and parse_int(cluster.last_leader_operation) or 0
```

Comment:
`last_leader_operation` supposed to be int, no need to cast it.

Reply:
will fix it.